src/main/views/viewManager.ts

```diff
@@ -189,8 +188,11 @@ export class ViewManager {
         // show the focused tab (or initial)
         if (focusedTuple && views.has(focusedTuple)) {
             const view = views.get(focusedTuple);
-            this.currentView = view!.name;
-            this.showByName(view!.name);
+            if (view) {
+                this.currentView = view.name;
+                this.showByName(view.name);
+                this.mainWindow.webContents.send(SET_ACTIVE_VIEW, view.tab.server.name, view.tab.type);
```

Comment:
Shouldn't this be sent as part of `showByName`? Why's it being sent separately

Reply:
`showByName` does not send this if the view is already - active, which is the case when the server name changes but not the URL. This is to ensure that it is sent no matter what. It doesn't do any harm to send it twice.